scrapy/extensions/feedexport.py

```diff
@@ -156,19 +156,29 @@ def _store_in_thread(self, file):
 
 class FTPFeedStorage(BlockingFeedStorage):
 
-    def __init__(self, uri):
+    def __init__(self, uri, active=False):
```

Comment:
I think `use_active_mode` is less likely to be miss-interpreted than `active` alone.

Reply:
Agreed, will change.